src/core_init_atmosphere/mpas_init_atm_cases.F

```diff
@@ -2854,7 +2854,7 @@ subroutine init_atm_case_gfs(block, mesh, nCells, nEdges, nVertLevels, fg, state
       character (len=StrKIND), pointer :: config_interface_projection
 
       real (kind=RKIND) :: truelat1, truelat2, xlonc, cone
-
+      real (kind=RKIND) :: temp
```

Comment:
`temp` is not the best variable name

Reply:
Agree. But I cannot figure out a better name.